### PR TITLE
Mood form

### PR DESCRIPTION
**Pull request by piperjarrett**
What does this branch do?
Adds a form to the homepage, allows user to click up to ten moods, then we find a match is clicked, they are redirected to a page that shows the dogs who have those same moods. 
How was it tested? 
No errors in the console and everything worked as it should be a link was clicked. 
Additional notes:
I added some CSS, we still will need to play with it some more.

### CODE REVIEW

**Review by forsethnico — APPROVED** [review]
Looks great! But I do think we could simplify the checkboxes into smaller groupings in maybe a dropdown for a better user experience.

**Review by Universal-Patois — APPROVED** [review]
Everything looks great! Good work Piper

File: src/components/App/App.css [hunk 1]
```diff
@@ -2,6 +2,9 @@ html {
   background-color: white;
   height: 100%;
 }
+body {
+  background-color: #F1AC88
```
Comment (letakeane) [inline src/components/App/App.css:6]: Don't forget the semicolon :D

File: src/components/App/App.tsx [hunk 2]
```diff
@@ -1,64 +1,101 @@
...
+import logo from "../../assets/canine-cupid_logo.png";
+import Dog from "../Dog/Dog";
 
 type appState = {
-  dogs: []
-  filteredDogs: []
-  favorites: number[]
-  error: string
-}
+  dogs: [];
```
Comment (letakeane) [inline src/components/App/App.tsx:15]: Remember, key/value pairs in an object are comma separated

File: src/components/App/App.tsx [hunk 3]
```diff
@@ -1,64 +1,101 @@
...
 
-  //In progress once form is created to get filter keywords
-  // filterDogsByTemperament = () => {
-  //   this.setState({...this.state, filteredDogs: [...this.state.filteredDogs, name]})
-  // }
+    this.setState({ ...this.state, filteredDogs: filterDogs });
+  };
 
   render() {
+    console.log(this.state.filteredDogs);
```
Comment (letakeane) [inline src/components/App/App.tsx:57]: Don't forget to remove console logs :)

File: src/components/MoodForm/MoodForm.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
+import { dogData } from "../../utilities/interfaces";
+import "./MoodForm.css";
+import { Link } from "react-router-dom";
+
+const MoodForm = ({
+  dogs,
+  filterDogsByTemperament,
+}: {
```
Comment (letakeane) [inline src/components/MoodForm/MoodForm.tsx:8]: Nice work building out these interfaces!

**Review by letakeane — COMMENTED** [review]
Mostly just finding small things, like formatting, commented out code, small syntax issues.

Nice work building out your TS!

**Pull request merged**